radix-engine/src/kernel/kernel_main_facade.rs

```diff
@@ -415,10 +419,15 @@ where
         self.scrypto_interpreter
     }
 
-    fn emit_wasm_instantiation_event(&mut self, code: &[u8]) -> Result<(), RuntimeError> {
-        KernelModuleMixer::on_wasm_instantiation(self, code)?;
+    fn emit_wasm_pre_instantiation_event(&mut self, code: &[u8]) -> Result<(), RuntimeError> {
+        KernelModuleMixer::before_wasm_instantiation(self, code)
+    }
 
-        Ok(())
+    fn emit_wasm_post_instantiation_event(
+        &mut self,
+        consumed_memory: usize,
+    ) -> Result<(), RuntimeError> {
+        KernelModuleMixer::after_wasm_instantiation(self, consumed_memory)
```

Comment:
I intend to remove these wasm-specific events in favor of the `ClientEventApi` (to be renamed to `ClientUnsafeApi`). Why do we need to emit post-instantiation event?

Reply:
Cool, thanks!

Can we please have consistent naming for the following? 
- `set_wasm_memory_consumption` -> `update_wasm_memory_usage`
- `wasm_memory_consumption` -> `update_wasm_memory_usage`
- `on_update_wasm_memory_usage` -> `on_update_wasm_memory_usage`  (no change)

Reply:
done